src/controllers/accounts/posts.js

```diff
@@ -55,6 +56,27 @@ const templateToData = {
 			const cids = await categories.getCidsByPrivilege('categories:cid', callerUid, 'topics:read');
 			return cids.map(c => `cid:${c}:uid:${userData.uid}:pids:votes`);
 		},
+		getTopics: async (sets, req, start, stop) => {
+			let pids = await db.getSortedSetRevRangeByScore(sets, start, stop, '+inf', '1');
+			pids = await privileges.posts.filter('topics:read', pids, req.uid);
+			const postObjs = await posts.getPostSummaryByPids(pids, req.uid, { stripTags: false });
+			return { posts: postObjs, nextStart: stop + 1 };
+		},
```

Comment:
The default `posts.getPostSummariesFromSet` call does not allow for `min` and `max` args, so I execute the same logic as that method, but call the `byScore` variant instead.

Not sure if there is a more elegant way you'd prefer.